extensions/resource-deployment/notebooks/bdc/2019/deploy-bdc-aks.ipynb

```diff
@@ -331,7 +331,7 @@
                 "os.environ[\"AZDATA_USERNAME\"] = mssql_username\n",
                 "os.environ[\"AZDATA_PASSWORD\"] = mssql_password\n",
                 "if os.name == 'nt':\n",
-                "    print(f'If you don\\'t see output produced by azdata, you can run the following command in a terminal window to check the deployment status:\\n\\tkubectl get pods -n {mssql_cluster_name} ')\n",
+                "    print(f'If you don\\'t see output produced by azdata, you can run the following command in a terminal window to check the deployment status:\\n\\t{os.environ[\"AZDATA_NB_VAR_kubectl\"]} get pods -n {mssql_cluster_name} ')\n",
```

Comment:
To be consistent it would nice to make the var all capitals (especially since it looks like the actual var you set is capitalized already)